docs/installation.md

```diff
@@ -16,11 +16,24 @@ composer require westacks/telebot
 
 If you are using Laravel, the library will self-register its ServiceProvider and Facade using Laravel's auto-discovery. Only Laravel version 5.5+ supported by the library.
 
+Tested working with 6.x.
+
 ### Publish Configuration File
 
 Open your terminal window and fire the following command to publish config file to your config directory:
 
 ```bash
 php artisan vendor:publish --provider="WeStacks\TeleBot\Laravel\TeleBotServiceProvider"
 ```
-Now you can find your bots config on `config/telebot.php` file.
\ No newline at end of file
+Now you can find your bots config on `config/telebot.php` file.
+
+### Optional \ Additional Configuration Options
+
+You may add the following to your .env file if you would prefer to edit the settings later from there.
+```
+ TELEGRAM_BOT_TOKEN=
```

Comment:
Token is required field to create a bot instance. It may confuse user if it writen under the heading 'Optional'